pkg/varlinkapi/containers.go

```diff
@@ -429,3 +444,21 @@ func (i *LibpodAPI) DeleteStoppedContainers(call ioprojectatomicpodman.VarlinkCa
 	}
 	return call.ReplyDeleteStoppedContainers(deletedContainers)
 }
+
+// GetAttachSockets ...
+func (i *LibpodAPI) GetAttachSockets(call ioprojectatomicpodman.VarlinkCall, name string) error {
+	runtime, err := libpodruntime.GetRuntime(i.Cli)
+	if err != nil {
+		return call.ReplyRuntimeError(err.Error())
+	}
+	ctr, err := runtime.LookupContainer(name)
+	if err != nil {
+		return call.ReplyContainerNotFound(name)
+	}
+	s := ioprojectatomicpodman.Sockets{
+		Container_id:   ctr.ID(),
+		Io_socket:      ctr.AttachSocketPath(),
+		Control_socket: filepath.Join(ctr.BundlePath(), "ctl"),
```

Comment:
Instead of making BundlePath public, it would probably be better to have a ctr.ControlSocketPath() that returns this

Reply:
i like that idea ... done